### PR TITLE
doomretro: update to 4.2.2.

### PR DESCRIPTION
**Pull request by benalb**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by Chocimier** [comment]
Please add changelog: https://raw.githubusercontent.com/bradharding/doomretro/master/releasenotes.md

**Comment by benalb** [comment]
done.

File: srcpkgs/doomretro/template [hunk 1]
```diff
@@ -9,5 +9,6 @@ short_desc="Refined DOOM source port"
 maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://www.doomretro.com"
+changelog="https://github.com/bradharding/doomretro/releases"
```
Comment (ericonr) [inline srcpkgs/doomretro/template:12]: I believe Chocimier intended for you to use https://raw.githubusercontent.com/bradharding/doomretro/master/releasenotes.md instead.

**Comment by benalb** [comment]
Fixed, and, in the meantime, upstream updated to 4.2.2.

**Pull request merged**